doc/source/geometric_manipulations.rst

```diff
@@ -0,0 +1,243 @@
+Geometric Manipulations
+========================
+
+
+
+
+Set-theoretic Methods
+~~~~~~~~~~~~~~~~~~~~~
+
+.. attribute::  GeoSeries.boundary
+
+  Returns a ``GeoSeries`` of lower dimensional objects representing
+  each geometries's set-theoretic `boundary`.
```

Comment:
To what extent is this overview of methods the same as what is listed in `reference.rst` ? I wonder if we should have it in both? (also in terms of keeping the docs up to date -> always having to update it at two places)